packages/common/src/types/fieldMatch.ts

```diff
@@ -1,4 +1,6 @@
 export type FieldValueSearchResult<T = unknown> = {
     search: string;
     results: T[];
+    cached: boolean;
```

Comment:
added new properties to this type.
This type is only used for this `filter search`